src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SceneFlameGraph.tsx

```diff
@@ -55,7 +60,12 @@ export class SceneFlameGraph extends SceneObjectBase<SceneFlameGraphState> {
 
       dataSubscription = newState.$data?.subscribeToState((newDataState) => {
         if (newDataState.data?.state === LoadingState.Done) {
-          timelineAndProfileApiClient.setLastTimeRange(newDataState.data.timeRange);
+          const lastTimeRange = newDataState.data.timeRange;
+
+          // For the "Function Details" feature only
+          timelineAndProfileApiClient.setLastTimeRange(lastTimeRange);
```

Comment:
We should also migrate the GitHub integration to Scenes at some point in the future.